.github/workflows/scripts/venv.sh

```
@@ -0,0 +1,36 @@
+#!/usr/bin/env sh
+
+set -e
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+# FIND Python binary based on environment
+if command -v python3 >/dev/null 2>&1; then
+    PYTHON_BIN=python3
+elif command -v python >/dev/null 2>&1; then
+    PYTHON_BIN=python
```

Shellcheck (0.11.0)

[warning] PYTHON_BIN appears unused. Verify use (or export if used externally).

(SC2034)